src/vaadin-chart-series.html

```diff
@@ -131,35 +128,38 @@
              * Note that you should always use [Polymer API](https://www.polymer-project.org/2.0/docs/devguide/model-data#array-mutation)
              * to mutate the values array in order to make the component aware of the
              * change and be able to synchronize it.
+             * @type {ChartSeriesValues}
              */
             values: {
               type: Array,
               value: () => []
             },
 
             /**
-             *  Value-axis minimum-value.
-             *  Sets the value to a series bound by 'unit' property.
-             *  Otherwise sets the value to the first series.
-             *  Undefined by default (determined from data).
+             * Value-axis minimum-value.
+             * Sets the value to a series bound by 'unit' property.
+             * Otherwise sets the value to the first series.
+             * Undefined by default (determined from data).
+             * @type {number | null}
              */
             valueMin: {
```

Comment:
`valueMin` has no default value so should include `| undefined`.

Reply:
Removed the `@type` to get the type from analyzer.